README.md

```diff
@@ -271,9 +271,11 @@ You can use `rustup` to install them both:
 rustup component add clippy --toolchain stable
 rustup component add rustfmt --toolchain nightly
 ```
-To run the linter tests, run the `check-clippy.sh` and `check-rustfmt.sh` scripts in the `.evergreen` directory:
+Our linter tests also use `rustdoc` to verify that all necessary documentation is present and properly formatted. `rustdoc` is included in the standard Rust distribution.
+
+To run the linter tests, run the `check-clippy.sh`, `check-rustfmt.sh`, and `check-rustdoc.sh` scripts in the `.evergreen` directory:
```

Comment:
This makes me wonder if maybe we should have a catch-all lint script for users to run instead of having them run so many different ones (although we'll still want to have separate ones so we can see which ones passed and which ones failed on evergreen). No need to make this script for this PR, though!

Reply:
SGTM, filed [RUST-652](https://jira.mongodb.org/browse/RUST-652) to do this in the future.